### PR TITLE
[WIP] Issue #749 Adding echo addon command

### PR DESCRIPTION
**Pull request by LalatenduMohanty**
Signed-off-by: Lalatendu Mohanty <lmohanty@redhat.com>

### CODE REVIEW

File: pkg/minishift/addon/command/echo_command.go [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+limitations under the License.
+*/
+
+package command
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/pkg/errors"
```
Comment (budhrg) [inline pkg/minishift/addon/command/echo_command.go]: It should be standard package `errors`.

File: pkg/minishift/addon/command/echo_command.go [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+func NewEchoCommand(command string) *EchoCommand {
+	defaultCommand := &defaultCommand{rawCommand: command}
+	echoCommand := &EchoCommand{defaultCommand}
+	defaultCommand.fn = echoCommand.doExecute
+	return echoCommand
+}
+
+func (c *EchoCommand) doExecute(ec *ExecutionContext) error {
+	// split off the actual 'echo' command. As we need to print rest of the string
+	cmd := strings.Replace(c.rawCommand, "echo ", "", 1)
```
Comment (hferentschik) [inline pkg/minishift/addon/command/echo_command.go]: You are missing the interpolation of the command

**Comment by hferentschik** [comment]
@LalatenduMohanty I fixed the remaining issues as part of pull request #766. I think we can close this one. WDYT?

**Comment by LalatenduMohanty** [comment]
@hferentschik sure.

**Comment by hferentschik** [comment]
Superset by pull request #766.

**Pull request closed without merging**